sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -201,8 +204,16 @@ def _wrapper_get_sfp_error_description(physical_port):
         except NotImplementedError:
             pass
     return None
-# Remove unnecessary unit from the raw data
 
+def _wrapper_reset_transceiver(physical_port):
+    if platform_chassis:
+        try:
+            return platform_chassis.get_sfp(physical_port).reset()
+        except NotImplementedError:
+            pass
+    return platform_sfputil.reset(physical_port)
```

Comment:
why do we need to reset the transceiver after insersion?

Reply:
This is required for all QSFP28/QSFP-DD modules that follow CMIS spec so that host can deterministically start accessing the I2C interface after 2 seconds delay of de-asserting the resetL.

Reply:
resetL means put the module out of reset, I thought that is the default behavior, this reset api is to first set the reset pin to high, and then to low. therefore, i am not sure you need this operation.

Reply:
as per spec, its the host which is responsible for bringing the module out of reset (even though it *may be* done automatically in hw), hence I think its better host software take explicit control of the module initialization rather than believing it will be done by hw. Do you see any problem?

Reply:
if you really want to make sure the module is out of reset, i suggest to add additional platform API to set resetL. In that case, if the module is already out of reset, then the api will do nothing. otherwise, the api will bring the module out of reset. current reset API is doing resetH, and then resetL which is not what the spec says.

Reply:
Ok, if platform is taking care, removing the reset